go.mod

```diff
@@ -41,7 +41,7 @@ require (
 	github.com/nic-at/rc0go v1.1.1
 	github.com/openshift/api v0.0.0-20200605231317-fb2a6ca106ae
 	github.com/openshift/client-go v0.0.0-20200608144219-584632b8fc73
-	github.com/oracle/oci-go-sdk v21.4.0+incompatible
+	github.com/oracle/oci-go-sdk/v46 v46.1.0 // indirect
```

Comment:
what's the reason of having indirect module?

Reply:
I think it was a mistake, should be fixed now (`go mod why` seemed to clean it up by itself).